panel/links.py

```diff
@@ -72,7 +74,7 @@ def assert_target_syncable(source, target, properties):
                 "It requires a live Python kernel to have an effect."
             )
 
-
+UnknownType = Any
```

Comment:
I use this to signal that I simply cannot figure out which types are allowed. I need your help @philippjfr .